nova/nova.go

```diff
@@ -798,11 +798,6 @@ func (c *Client) AttachVolume(serverId, volumeId, device string) (*VolumeAttachm
 	}
 	url := fmt.Sprintf("%s/%s/%s", apiServers, serverId, apiVolumeAttachments)
 	err := c.client.SendRequest(client.POST, "compute", "v2", url, &requestData)
-	if errors.IsNotFound(err) {
-		return nil, errors.NewNotImplementedf(
-			err, nil, "the server does not support attaching volumes",
```

Comment:
What is the way that we could detect if the server just doesn't support volume attachment at all?
Is there a different error we should be handling?

Reply:
> Is there a different error we should be handling?

Not that I can see. Just query the extensions API endpoint first if needs be.